### ISSUE TITLE
Registry of  functional profiles?

### ISSUE THREAD

**Issue by pchampin** [w3c/dx-connegp#74]
[Section 8](https://w3c.github.io/dx-connegp/connegp/#functional-profiles) mentions a "a register of profile objects which currently contains just the 4 instances above from [Figure 2](https://w3c.github.io/dx-connegp/connegp/#fig-table-profiles)."

Readers may expect (especially because of the word "currently") to be able to register their own functional profile into that register, just like with IANA registries. Note that W3C now has a [Registry Track](https://www.w3.org/2023/Process-20231103/#registries).

Do we want to use the Registry Track to define formal registration process to augment the table in Table 2. If not, we should probably reword Section 8 to defuse this kind of expectation.

**Comment by nicholascar** [comment]
Yes, we do want a registry of Functional Profiles. We've seen a couple more FPs be considered recently (file name endings, not QSA or HTTP to indicate Media Types; OGC API implementation etc.),  so we will likely see more still, at a very slow growth rate.

**Comment by pchampin** [comment]
> Yes, we do want a registry of Functional Profiles. 

Ok, so we should probably start a new deliverable on the [Registry Track](https://www.w3.org/2023/Process-20231103/#registries).
